SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialAuthorityTest/SpatialAuthorityTest.cpp

```diff
@@ -29,13 +29,85 @@ ASpatialAuthorityTest::ASpatialAuthorityTest()
 	Description = TEXT("Test HasAuthority under multi-worker setups. It also ensures it works in Native");
 
 	Server1Position = FVector(-250.0f, -250.0f, 0.0f);
-	Server2Position = FVector(-250.0f, 250.0f, 0.0f);
+	Server2Position = FVector(250.0f, -250.0f, 0.0f);
```

Comment:
Can you update the test description above with the details of these changes

Reply:
Sure, done.